examples/models/llava/export_llava.py

```diff
@@ -296,6 +297,12 @@ def main():
         action=BooleanOptionalAction,
         help="Use sdpa_with_kv_cache custom op in LLava text model.",
     )
+    parser.add_argument(
+        "--max-context-len",
+        default=768,
```

Comment:
i think we should make this a required arg. this default could be a footgun as well